packaging/src/main/assembly/bin.xml

```diff
@@ -103,7 +103,7 @@
       <excludes>
         <exclude>org.apache.hadoop:*</exclude>
         <exclude>org.apache.hive:hive-jdbc:jar:standalone</exclude>
-        <exclude>org.apache.httpcomponents:*</exclude>
+        <!--exclude>org.apache.httpcomponents:*</exclude-->
```

Comment:
instead of dropping the exclusion that is used in case full project build
> ls -la packaging/target/apache-hive-4.0.1-bin/apache-hive-4.0.1-bin/hcatalog/share/webhcat/svr/lib | grep http
httpclient-4.5.13.jar
httpcore-4.4.13.jar
````
mvn clean install -Pdist -DskipTests -Dmaven.javadoc.skip=true -DcreateChecksum=true
````
maybe set `useStrictFiltering` to `false`?

Reply:
hi @basapuram-kumar, WDYT? Can we do the above change here `set useStrictFiltering to false`